libs/application/templates/document-provider-onboarding/src/lib/DocumentProviderOnboardingTemplate.ts

```diff
@@ -56,6 +56,7 @@ const dataSchema = z.object({
   administrativeContact: contact,
   technicalContact: contact,
   helpDesk: helpDeskContact,
+  endPoint: z.string().nonempty(),
```

Comment:
should this be z.url().nonempty() ?

Reply:
It seems that z.url() does not exist... Can i add it or am i missing something ?